config/rbac/role.yaml

```diff
@@ -1,10 +1,8 @@
-
----
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
   creationTimestamp: null
-  name: manager-role
+  name: {{ include "actions-runner-controller.managerRoleName" . }}
```

Comment:
This seems messed up. This is a resource loaded by ARC's kustomize config, which has no access to these Helm template expressions

Reply:
good call - right. fixed in https://github.com/actions-runner-controller/actions-runner-controller/pull/1268/commits/9c0c9add18c8fa1f376162081c27f03da30d3e8f